### PR TITLE
#194 Refactored extensions-authzforce

### PR DESCRIPTION
**Pull request by habibimoiz**
#194 Merging into feature branch. Changes include:
 - refactored extensions-authzforce-attribute
 - extensions-policy-decision-point-service
 - updated velocity templates for generated files.

**Upcoming changes:**
- Write migrations for the refactored changes to the extensions-security modules.

### CODE REVIEW

File: extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java [hunk 1]
```diff
@@ -49,18 +39,17 @@
...
 
-import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.github.benmanes.caffeine.cache.Cache;
-import com.github.benmanes.caffeine.cache.Caffeine;
-
-import oasis.names.tc.xacml._3_0.core.schema.wd_17.AttributeDesignatorType;
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
```
Comment (chang-annie) [inline extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java:44]: A: replace wildcard import with explicit imports

File: extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java [hunk 2]
```diff
@@ -156,22 +145,22 @@ public <AV extends AttributeValue> AttributeBag<AV> get(AttributeFqn attributeGU
...
      * @param id
      *            attribute id
      * @param subject
      *            subject name
      * @return collection of attributes or null if none found
      */
-    public Collection<com.boozallen.aiops.security.authorization.policy.AttributeValue<?>> getAiopsAttributeByIdAndSubject(String id,
-            String subject) {
+    public Collection<com.boozallen.aissemble.security.authorization.policy.AttributeValue<?>> getAissembleAttributeByIdAndSubject(String id,
+                                                                                                                               String subject) {
```
Comment (chang-annie) [inline extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java:157]: A: looks like extra spaces added here

Comment (habibimoiz) [reply extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java:166]: I think GH is making the formatting look a bit off, in the IDE it looks more pleasing.

File: extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java (hunk 1 above)
Comment (habibimoiz) [reply extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java]: Applied the changes.

**Review by chang-annie — APPROVED** [review]
LGTM

**Pull request merged**